pypika/tests/test_selects.py

```diff
@@ -30,6 +31,15 @@ def test_empty_query(self):
 
         self.assertEqual('', str(q))
 
+    def test_select_no_from(self):
+        q = Query.select(1)
+
+        self.assertEqual('SELECT 1', str(q))
+
+    def test_select_no_from_with_field_raises_exception(self):
```

Comment:
Just wondering - will this stop queries such as `select @@version` in MySQL?

Reply:
Don't think that would've been possible to begin with.